src/main/headless.ts

```diff
@@ -120,7 +126,19 @@ class Headless {
     const body = JSON.stringify({
       PrivateKeyString: privateKey,
     });
-    return this.retriableFetch("set-private-key", body);
+    const httpResult = await this.retriableFetch("set-private-key", body);
```

Comment:
Just curious, do we still need `set-private-key` endpoint?

Reply:
At this time, yes. It is for mining. To remove it, https://github.com/planetarium/NineChronicles.Headless/pull/413 pull request should be merged.

Reply:
👌